content/guides/integration-guides/cinder-compatible-SODA-API/_index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Cinder Compatible 
+menuTitle: "SODA API"
+description: "A guide for enable Cinder Compatible SODA API."
+weight: 1
+disableToc: false
+tags: ["Cinder Compatible SODA API"] 
+---
+## Install SODA
+
+### Pre-requisite 
+* Ubuntu 18.04
+* Ram 16GB
+* CPU minimum 4Core
+
+Install SODA using ansible script following the steps given in the [link](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)
+
+Or install using [devsds script](https://docs.sodafoundation.io/guides/developer-guides/file-share/)
+
+
+After installation of SODA check all the services are up before installation of Openstack.
```

Comment:
If there are steps already in the documentation to do this please link, else add the commands here.